### ISSUE TITLE
Typing errors in misc/scaling/service.yaml

### ISSUE THREAD

**Issue by philippemerle** [xlab-si/xopera-examples#37]
[misc/scaling/service.yaml](https://github.com/xlab-si/xopera-examples/blob/master/misc/scaling/service.yaml) contains some typing errors.

The `radon.nodes.aws.AwsLambda` node type defines:
1.  the `tosca.interfaces.node.lifecycle.Standard` interface instead of refining the `Standard` interface. Here the interface should be renamed.
2.  a new interface named `radon.interfaces.scaling` without specifying its interface type. Here a new interface type should be defined and the `type` of the `radon.interfaces.scaling` interface should be set. 

The operation name of a `call_operation` activity uses the `<interface_name>.<operation_name>` notation, c.f. Section 3.6.23.3 of [TOSCA 1.3](https://docs.oasis-open.org/tosca/TOSCA-Simple-Profile-YAML/v1.3/TOSCA-Simple-Profile-YAML-v1.3.html). According to this TOSCA rule, the `call_operation: radon.interfaces.scaling.scale_down` activity could be interpreted ambiguously, e.g., 
1. the `interface_name` is equals to `radon` and the `operation_name` is equals to `interfaces.scaling.scale_down`, 
2. the `interface_name` is equals to `radon.interfaces` and the `operation_name` is equals to `scaling.scale_down`,
3. the `interface_name` is equals to `radon.interfaces.scaling` and the `operation_name` is equals to `scale_down`.

To avoid this ambiguity, I would like to recommend to rename the `radon.interfaces.scaling` interface of the `radon.nodes.aws.AwsLambda` node type to `Scaling` and to rewrite the activity as `call_operation: Scaling.scale_down`.
The same issue also appears in the `call_operation: radon.interfaces.scaling.scale_up` activity.

To summarize, [misc/scaling/service.yaml](https://github.com/xlab-si/xopera-examples/blob/master/misc/scaling/service.yaml) should contain:
```
...
interface_types:
  radon.interfaces.Scaling:
    derived_from: tosca.interfaces.Root
    operations:
      scale_down:
        description: scale down
      scale_up:
        description: scale up

node_types:
  radon.nodes.aws.AwsLambda:
    derived_from: tosca.nodes.SoftwareComponent
    interfaces:
      Standard:
        operations:
          create: playbooks/create.yaml
          delete: playbooks/delete.yaml
      Scaling:
        type: radon.interfaces.Scaling
        operations:
          scale_down: playbooks/scale_down.yaml
          scale_up: playbooks/scale_up.yaml
...
```

**Comment by anzoman** [comment]
Hi @philippemerle thanks, interesting interpretation and description! I agree, according to TOSCA we should definitely rename the ambiguous call operation activity definition.
